gems/pending/util/miq-process.rb

```diff
@@ -127,24 +127,7 @@ def self.processInfo(pid = nil)
   end
 
   def self.command_line(pid)
-    case Sys::Platform::IMPL
-    when :mswin, :mingw
```

Comment:
@Fryguy @djberg96 @gmcculloug I didn't test windows but [sys-proctable supports it](https://github.com/djberg96/sys-proctable/blob/1e84f9c20783d282220cef998c4feb963bdbbc21/lib/windows/sys/proctable.rb#L168-L172) but I have:
* no idea if they return the same value
* if we care, I don't see MiqProcess.command_line being run on windows machines.